src/components/state-details.js

```diff
@@ -581,6 +578,23 @@ export default function StateDetailsPage ({ location, data }) {
                 </>
               )}
 
+              <div className="card">
+                <div className="card-body">
+                  <h5 class="card-title">
+                    <a href="https://bit.ly/dirty-power-plants-usa" target="_blank" rel="noreferrer">
+                      See a map of dirty power plants in the US
+                    </a>
+                  </h5>
+                  <a href="https://bit.ly/dirty-power-plants-usa" target="_blank" rel="noreferrer">
+                    <img
+                      className="img-fluid"
+                      src={PowerPlantMap}
+                      alt="Map showing coal, gas and oil power plants in the United States"
```

Comment:
This image is essentially decorative considering the link text above, so I think we can use `alt=""` to indicate that.